src/components/BioCard.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { GridContainer, Card, CardGroup } from "@trussworks/react-uswds";
+
+function BioCard(props) {
+  const { name, photo } = props;
+  return (
+    <CardGroup>
```

Comment:
Everything is looking good over all! 

- I'm wondering how CardGroup interacts with Grid. Can CardGroup replace Grid? 
- Also would it make sense to have CardGroup in your Team.js file around the map that returns multiple BioCards, instead of being around just one instance of a Card?